pages/bitmark-references/node-setup/bitmark-node-setup.md

```diff
@@ -9,154 +9,116 @@ folder: bitmark-references/node-setup
 
 #  Bitmark Node Installation
 
-The [Bitmark Quickstart](../../learning-bitmark/quick-start) includes a tutorial on [installing the Bitmark Property System using docker](../../learning-bitmark/quick-start/simple-solution-for-node-setup.md). That setup is intended for beginners who want to make it fast and easy to run the Bitmark Property System using a `docker` container that wraps all components and settings. Developers, or people who want to study the main program, may instead choose to run `bitmarkd` or `recorderd` services directly. 
-
 A full-node of bitmarkd consists of a `bitmarkd`, a `recorderd`, a payment system, a `litecoind`, and a `bitcoind`. The `bitmarkd` service  verifies and records transactions on the Bitmark blockchain while the `recorderd` service computes the Bitmark proof-of-work algorithm that allows nodes to compete to win blocks on the Bitmark blockchain. 
 
 This tutorial explains how to set up the `bitmarkd` and/or the `recorderd` services by hand.
 
 ## Prerequisites
 
-* You must install the go language package for your system.
-* You must configure environmental variables for the go system.
-* You must install the ZMQ4 and Argon2 libraries.
-
-## Installing the Prerequisite Packages
++ You must have and know your **public IP** address.
++ You must have **a Bitcoin and a Litecoin address** for receiving mining rewards.
++ Your environment must be open for incoming connections through port 2136.
++ Hardware minimum requirements:
+    - memory >= 2 GB
+    - free disk space >= 20 GB
+    - broadband Internet connection with upload speeds of at least 400 kilobits (50 kilobytes) per second
 
-Installing the required packages requires specific commabds on Debian, FreeBSD, macOS, and Ubuntu.
+## Installing a Bitmark Node and Generating Required Keys and Certificates
 
-### Installing Packages on Debian
+A Bitmark node can be installed differently, depending on the operating system. Installations on FreeBSD, Ubuntu, and macOS can be done through pre-built packages.
 
```

Comment:
Not sure what "installed differently" means. Perhaps something like "Bitmark nodes can be installed on a variety of different operating system, with pre-built packages available for FreeBSD, Ubuntu, and MacOS."